examples/demo-rollup/tests/module_system_integration_tests.rs

```diff
@@ -0,0 +1,66 @@
+mod test_helpers;
+use std::net::SocketAddr;
+
+use borsh::BorshSerialize;
+use demo_stf::app::DefaultPrivateKey;
+use demo_stf::runtime::RuntimeCall;
+use jsonrpsee::core::client::ClientT;
+use jsonrpsee::http_client::{HttpClient, HttpClientBuilder};
+use sov_modules_api::default_context::DefaultContext;
+use sov_modules_api::transaction::Transaction;
+use sov_modules_api::{PrivateKey, Spec};
+use test_helpers::start_rollup;
+
+struct TestClient {
```

Comment:
This is a generic code that could be reused by other rollups. Let's rename it to `SimpleRolluoClient` or something similar and move it to `full-node` or even `rollup-interface`?

Reply:
I moved it to the `sov-sequencer` crate since...
1. `rollup-interface` doesn't currently take a dependency on jsonrpsee
2. It's implementation is coupled with the sequencer rpc implementation